public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -177,6 +180,10 @@ export function formValuesToRulerGrafanaRuleDTO(values: RuleFormValues): Postabl
         exec_err_state: execErrState,
         notification_settings: notificationSettings,
         metadata,
+        missing_series_evals_to_resolve: missingSeriesEvalsToResolve
+          ? Number(missingSeriesEvalsToResolve)
+          : // API uses 0 value to reset, as `missing_series_evals_to_resolve` cannot be 0
```

Comment:
Can we say that with '0' we reset to the default value, that is '2' ?(https://github.com/grafana/grafana/pull/101184)